pyproject.toml

```diff
@@ -67,3 +79,15 @@ commands =
     mkdir -p docs/build
     sphinx-build -b html -Ean -j 2 -w build/sphinx-errors \
         docs/source docs/build
+
+[testenv:release]
+basepython=python3.8
+deps = pep517
+whitelist_externals =
+    cp
+    rm
+commands =
+    rm -rf {toxinidir}/dist
+    cp --no-target-directory --recursive {distdir} {toxinidir}/dist; # copy the wheel built by tox-wheel
```

Comment:
@adiroiban this is where the magic happens - I can now run `pipx run tox -e py38-test-twlatest,release && twine upload dist/*` and tox will build the whl and *that whl file* will be installed and tested and *that whl file* will be uploaded to pypi

Reply:
I am -1 for any magic stuff :) When I read that line it makes no sense to me.... so I guess that a comment is required.

What is wrong with using the `python -m pep517.build --source --binary --out-dir dist/ .` command to generate the source dist and binary wheel?

Also, I think that `tox -e release` is the wrong environment name. Maybe it should be `tox -e create-dist` or something like that.

The actual release/publish should be a separate step as it required a repo URL + credentials.

Reply:
> What is wrong with using the python -m pep517.build --source --binary --out-dir dist/ . command to generate the source dist and binary wheel?

I want to test the actual whl file that gets released

Reply:
> Also, I think that tox -e release is the wrong environment name. Maybe it should be tox -e create-dist or something like that.

I'm happy with either name, can you push a fix with what you think is best?

Reply:
> @adiroiban this is where the magic happens 

Maybe magic was the wrong word. This line is the culmination/apex of the PR that everything else was building to. It allows us to create an artifact, test it and then distribute it

Reply:
@adiroiban pinging you on this change. What sort of comment would be best here?

Reply:
Hi... I got it now :)
There was the comment that the wheel is created by tox, but I have not seen any tox command to create it.

But I now know what it going on here. For every env, tox will automatically create a wheel and install it in the dedicated virtualenv.